english/game-on/game_on.md

```diff
@@ -0,0 +1 @@
+Suddenly he appears and passes the ball to me. I feel like an [nba](www.nba.com)star. I fell unstoppable.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.